src/services/package-manager/batch-import-export-service.ts

```diff
@@ -118,20 +124,37 @@ class BatchImportExportService {
         return batchImportExportApi.findVariablesWithValuesByPackageKeysAndVersion(variableExportRequest)
     }
 
-    private buildBodyForImport(file: string, configs: AdmZip): FormData {
+    private buildBodyForImport(configs: AdmZip, variablesManifests: VariableManifestTransport[]): FormData {
         const formData = new FormData();
+        const readableStream = this.getReadableStream(configs);
 
-        formData.append("file", fs.createReadStream(file));
+        formData.append("file", readableStream, {filename: "configs.zip"});
```

Comment:
I'm curious on why the `filename` is required?

Reply:
When I try without it, this [error](https://celonis.datadoghq.com/logs?query=892c531d-a9be-41d1-bdcb-c34f43008a15%20&cols=core_host%2Ccore_service&event=AgAAAY3Q752c3NTq7AAAAAAAAAAYAAAAAEFZM1E3NmJWQUFDTkx6NDRkeURYWXdBbQAAACQAAAAAMDE4ZGQwZWYtYzgzNS00YzA1LTk1ZDktNWYwYjAwNzUxZTY0&index=%2A&messageDisplay=inline&stream_sort=desc&viz=stream) is being thrown